alibi_detect/od/ensemble.py

```diff
@@ -0,0 +1,26 @@
+import imp
+from typing import Optional, List
+from alibi_detect.od.base import OutlierDetector
+from alibi_detect.od.aggregation import BaseTransform, AverageAggregator, PValNormaliser
+from alibi_detect.od.processor import BaseProcessor
+
+
+class Ensemble(OutlierDetector):
+    def __init__(
+            self,
+            detectors: List[OutlierDetector], 
+            aggregator: Optional[BaseTransform] = AverageAggregator(), 
+            normaliser: Optional[BaseTransform] = PValNormaliser(),
+            processor=BaseProcessor):
+
+        self.detectors = detectors
+        self.normaliser = normaliser
+        self.aggregator = aggregator
+        self.processor = processor(detectors=detectors)
+
+    def fit(self, X):
+        for detector in self.detectors:
+            detector.fit(X)
```

Comment:
Even if this is done in parallel, `fit` times vary wildly from basically zero (non-parametric e.g. `knn`) and lengthy model fitting phases. Wonder if worth investigating ensembling fitted detectors so the fitting gets outsourced to the user in whatever way they want to do it?

Reply:
Minor tangential thought; if we are not going to have an `Ensemble.fit` method, it might still be worth looping through all detectors and checking for a `fitted` attribute or something? Rather than leaving it until `score` is called to find out...